lib/maze/server.rb

```diff
@@ -2,10 +2,11 @@
 
 require 'json'
 require 'webrick'
-require_relative './servlet'
-require_relative './log_servlet'
 require_relative './logger'
 require_relative './request_list'
+require_relative './servlets/command_servlet'
```

Comment:
Is this redundant as it's already imported above: https://github.com/bugsnag/maze-runner/pull/332/files#diff-8f360d1c0c26c88707e927813ccd865245e4596f406eae23809d354b2ed5d53cR25?

Reply:
Yes, I think you're right.  I think the order of requires matters and I got it wrong in `maze_runner` at first, leading to me adding these unnecessarily.  It just means adding some requires to the unit test.